spark/src/main/scala/org/apache/spark/Plugins.scala

```diff
@@ -101,14 +101,16 @@ class CometDriverPlugin extends DriverPlugin with Logging with ShimCometDriverPl
    * unified memory manager.
    */
   private def shouldOverrideMemoryConf(conf: SparkConf): Boolean = {
-    conf.getBoolean(CometConf.COMET_ENABLED.key, true) && (
-      conf.getBoolean(
-        CometConf.COMET_EXEC_SHUFFLE_ENABLED.key,
-        CometConf.COMET_EXEC_SHUFFLE_ENABLED.defaultValue.get) ||
-        conf.getBoolean(
-          CometConf.COMET_EXEC_ENABLED.key,
-          CometConf.COMET_EXEC_ENABLED.defaultValue.get)
-    ) && CometSparkSessionExtensions.cometUnifiedMemoryManagerEnabled(conf)
+    val comet_enabled = conf.getBoolean(CometConf.COMET_ENABLED.key, true)
```

Comment:
This code was recently updated in main to remove the hard-coded default:

```suggestion
    val comet_enabled = conf.getBoolean(CometConf.COMET_ENABLED.key, CometConf.COMET_ENABLED.defaultValue.get)
```